src/ast/mod.rs

```diff
@@ -7357,6 +7354,99 @@ impl Display for UtilityOption {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct ShowStatementOptions {
```

Comment:
For the added structs/enums, since they're public can we add a short description mentioning what they are? we could probably also link to one of the snowflake show docs to serve as a generic reference for someone that comes across it